rubocop.yml

```diff
@@ -15,6 +15,75 @@ AllCops:
 Rails:
   Enabled: true
 
+Layout/ClassStructure:
+  Enabled: true
+  Categories:
+    module_inclusion:
+      - extend
+      - include
+      - prepend
+    callbacks:
+      - before_validation
+      - after_validation
+      - before_save
+      - around_save
+      - before_create
+      - around_create
+      - after_create
+      - before_update
+      - around_update
+      - after_update
+      - after_save
+      - before_destroy
+      - around_destroy
+      - after_destroy
+      - after_commit
+      - after_rollback
+      - after_initialize
+      - after_find
```

Comment:
These aren't in the desired order are they?  Correct me if I'm wrong, but, this is just stating that these callbacks should all be grouped together within the class.

Reply:
it would be kinda cool if we could specify an order to the individual level.

Reply:
>These aren't in the desired order are they? 

They are ordered in the chronological way they happened:
https://guides.rubyonrails.org/active_record_callbacks.html#available-callbacks

> Correct me if I'm wrong, but, this is just stating that these callbacks should all be grouped together within the class.

Yes, if it is a **callback** should be set in the right order and stay together so we know always where all of them are defined.